src/index.html

```diff
@@ -19,9 +19,402 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="images/logo.png"
```

Comment:
The 'href' attribute in the anchor tag should point to a valid URL, not an image path. Consider using a URL that makes sense for navigation, such as the homepage or a relevant section of the site.